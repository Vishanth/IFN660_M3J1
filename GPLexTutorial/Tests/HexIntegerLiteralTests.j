0xA_0_5L
0XA_0_BL
0xFL
0X1L
