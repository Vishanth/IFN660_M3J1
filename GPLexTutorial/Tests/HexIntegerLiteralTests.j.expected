INTEGER LITERAL ()
INTEGER LITERAL ()
INTEGER LITERAL ()
INTEGER LITERAL ()
EOF